### PR TITLE
Making the `run` command documentation more explicit

### PR DESCRIPTION
**Pull request by Nameless-123**
By: Ghoulboy

### CODE REVIEW

File: docs/carpet/script/CarpetExpression.html [hunk 1]
```diff
@@ -268,7 +268,11 @@ <h3><code>print(expr)</code></h3>
  <h3><code>logger(expr)</code></h3>
  <p>Prints the message to system logs, and not to chat.</p>
  <h3><code>run(expr)</code></h3>
- <p>Runs a command from the string result of the <code>expr</code> expression, and returns its success count</p>
+ <p>Runs a vanilla command <code>expr</code> and returns its success count</p>
+ <pre>
+ run('fill 1 1 1 10 10 10 air') -> 10 // 10 block were filled, this operation was successful 10 times out of a possible 1000 blocks volume
```
Comment (gnembon) [inline docs/carpet/script/CarpetExpression.html]:
```suggestion
 run('fill 1 1 1 10 10 10 air') -&gt; 10 // 10 block were filled, this operation was successful 10 times out of a possible 1000 blocks volume
```
its an html

File: docs/carpet/script/CarpetExpression.html [hunk 2]
```diff
@@ -268,7 +268,11 @@ <h3><code>print(expr)</code></h3>
...
+ run('give @s stone 4') -> 1 // this operation was successful once
```
Comment (gnembon) [inline docs/carpet/script/CarpetExpression.html]:
```suggestion
 run('give @s stone 4') -&gt; 1 // this operation was successful once
```
same

File: docs/carpet/script/CarpetExpression.html (hunk 1 above)
Comment (Nameless-123) [reply docs/carpet/script/CarpetExpression.html]: thanks gnembon, I had no clue

File: docs/carpet/script/CarpetExpression.html (hunk 2 above)
Comment (Nameless-123) [reply docs/carpet/script/CarpetExpression.html]: thanks again man!

**Comment by Nameless-123** [comment]
I had no clue what closing did. I wont until its on the scarpet page

**Comment by gnembon** [comment]
I applied that change to the javadocs but will merge it here as well, so its updated

**Pull request merged**